### PR TITLE
events added

### PR DESCRIPTION
**Pull request by debjit-bhowal-rs**
- TokenCreationStarted
- TokenCreationCompleted
- TokenCreationFailed
- TokensClaimedFromNeuron
- TokensTransferred

### CODE REVIEW

File: ssr/src/utils/event_streaming/events.rs [hunk 1]
```diff
@@ -726,3 +733,153 @@ impl ProfileViewVideo {
...
+
+            // token_creation_completed - analytics
+            send_event(
+                "token_creation_completed",
+                &json!({
+                    "user_id": user_id,
+                    "canister_id": canister_id,
+                    "token_name": sns_init_payload.token_name,
+                    "token_symbol": sns_init_payload.token_symbol,
+                    "name": sns_init_payload.name
```
Comment (komal-rs) [inline ssr/src/utils/event_streaming/events.rs:794]: can we also send the description here ?

File: ssr/src/utils/event_streaming/events.rs [hunk 2]
```diff
@@ -726,3 +733,153 @@ impl ProfileViewVideo {
...
+            let details = user.details;
+
+            let user_id = details.principal;
+            let canister_id = user.canister_id;
+
+            // token_creation_completed - analytics
+            send_event(
+                "token_creation_completed",
+                &json!({
+                    "user_id": user_id,
```
Comment (komal-rs) [inline ssr/src/utils/event_streaming/events.rs:790]: can we also send the token img url or base64 ?

File: ssr/src/utils/event_streaming/events.rs (hunk 1 above)
Comment (debjit-bhowal-rs) [reply ssr/src/utils/event_streaming/events.rs:794]: done

File: ssr/src/utils/event_streaming/events.rs (hunk 2 above)
Comment (debjit-bhowal-rs) [reply ssr/src/utils/event_streaming/events.rs:790]: i added it as a base64 string, in the field "logo"

**Review by komal-rs — APPROVED** [review]

**Pull request merged**